README.md

```diff
@@ -326,17 +329,13 @@ Currently for AOSP browsers 4.0.x - 4.1.x, the panes do not animate smoothly whe
 
 ### Requirements
 
-* [Node.js 0.10.x/npm](http://nodejs.org/download/)
+* [Node.js v4.x LTS + NPM v2.x](https://nodejs.org/en/download/) (Mobify recommends [NVM](https://github.com/creationix/nvm) for installing Node + NPM)
```

Comment:
I'll tighten up the docs to explicitly mark node v4 and npm v2 for compatibility. This requirement is only for local dev (to ensure that requirejs configs are correct for the examples and testing), and so that our README example is correct.